src/views/footnote.ts

```diff
@@ -57,32 +62,77 @@ export class FootnoteView extends BaseNodeView<Trackable<FootnoteNode>> {
       marker.innerText = fn.labels.get(id) || ''
       marker.addEventListener('mousedown', (e) => this.handleMarkerClick(e))
     }
-    const deleteBtn = document.createElement('span')
-    deleteBtn.classList.add('delete-icon')
-    deleteBtn.innerHTML = deleteIcon
-    deleteBtn.addEventListener('mousedown', (e) => this.handleDeleteClick(e))
+    const container = findFootnotesContainerNode(this.view.state.doc, pos)
+
+    let scrollBtn: HTMLElement | null = null
+    if (container.node.type !== schema.nodes.table_element) {
+      // Only create the scroll button if the parent is not a table
+      scrollBtn = this.createButton(
+        'scroll-icon',
+        scrollIcon,
+        this.handleMarkerClick
+      )
+    }
+    const deleteBtn = this.createButton(
+      'delete-icon',
+      deleteIcon,
+      this.handleDeleteClick
+    )
 
     this.dom.innerHTML = ''
     this.dom.classList.value = ''
     this.dom.classList.add('footnote')
     this.dom.classList.add(...getChangeClasses(this.node.attrs.dataTracked))
-    this.dom.appendChild(marker)
-    this.contentDOM && this.dom.appendChild(this.contentDOM)
-    this.dom.appendChild(deleteBtn)
+    this.dom.append(
+      marker,
+      ...(this.contentDOM ? [this.contentDOM] : []),
+      ...(scrollBtn ? [scrollBtn] : []), // Only append scrollBtn if it's created
+      deleteBtn
+    )
+  }
+  // Helper function to create buttons
+  private createButton = (
+    className: string,
+    icon: string,
+    eventHandler: (e: Event) => void
+  ): HTMLElement => {
+    const btn = document.createElement('span')
+    btn.classList.add(className)
+    btn.innerHTML = icon
+    btn.addEventListener('mousedown', eventHandler)
+    return btn
   }
 
   handleMarkerClick = (e: Event) => {
     e.preventDefault()
     e.stopPropagation()
+
     const id = this.node.attrs.id
     const fn = getFootnotesElementState(this.view.state, id)
+
     if (!fn) {
       return
     }
+
     for (const [node, pos] of fn.inlineFootnotes) {
       if (node.attrs.rids.includes(id)) {
         const tr = this.view.state.tr
         const selection = NodeSelection.create(this.view.state.doc, pos)
+        const targetElement = this.view.domAtPos(pos).node as HTMLElement
+        const markerElement = targetElement.querySelector(
+          '.footnote-marker'
+        ) as HTMLElement
+        // Ensure the marker exists and add highlight class for 3 seconds
+        // A grey bar fades in and out on the line where the marker is present to help user find marker
+        if (markerElement) {
+          markerElement.classList.add('highlight-footnote-marker')
+
+          setTimeout(
+            () => markerElement.classList.remove('highlight-footnote-marker'),
+            3000
+          )
+        }
```

Comment:
I think we should move this logic to `handleScrollToSelection` as well.

Reply:
done